src/components/book.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { removeBookAction } from './redux/books/books';
+
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Please don't disable any unpassed Linters rule so Kindly add PropTypes to this component because you are passing props to it. You can check this[ link](https://www.freecodecamp.org/news/how-to-use-proptypes-in-react/) for more info about how to do it.